app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/components/Navigation.kt

```diff
@@ -31,7 +32,7 @@ fun Navigation(
     albums: List<Album>,
     artists: List<Artist>,
     playlists: List<Playlist>,
-    recentlyPlayedSongs: Playlist,
+    recentlyPlayedSongs: List<Song>,
```

Comment:
What's wrong with a playlist here? I would rather stick to the playlist here because it does maintain a list of songs anyway.

Reply:
added data to playlist, now showing recently played using Playlist